lib/ts/recipe/emailverification/emailVerifiedClaim.ts

```diff
@@ -0,0 +1,51 @@
+import { SessionClaimValidator, BooleanClaim } from "../session";
+import { RecipeInterface } from "./types";
+
+export class EmailVerifiedClaimClass extends BooleanClaim {
+    constructor(getRecipeImpl: () => RecipeInterface) {
+        super({
+            id: "st-ev",
+            refresh: async (userContext) => {
+                await getRecipeImpl().isEmailVerified({
+                    userContext,
+                });
+            },
+        });
+
+        this.validators = {
+            ...this.validators,
+            isValidated: (minRefetchDelayInSeconds: number = 10, getRedirectPath?: () => string | Promise<string>) => ({
+                id: this.id,
+                refresh: this.refresh,
+                shouldRefresh: (payload, userContext) => {
+                    const value = this.getValueFromPayload(payload, userContext);
+                    return (
+                        value === undefined ||
+                        (value === false &&
+                            this.getLastFetchedTime(payload, userContext)! <
+                                Date.now() - minRefetchDelayInSeconds * 1000)
```

Comment:
```suggestion
                                (Date.now() - minRefetchDelayInSeconds * 1000))
```

Reply:
U just prefer to add brackets in situations like this.. just in case

Reply:
Prettier removes it.